### ISSUE TITLE
Build Failing

### ISSUE THREAD

**Issue by chris75898** [googlecodelabs/photos-sharing#15]
Running the following commands:
`git clone https://github.com/googlecodelabs/photos-sharing.git`
`cd photos-sharing`
`flutter run`

Produces the below results. The first thing I tried was migrate to AndroidX but it had no impact.
```
Using hardware rendering with device AOSP on IA Emulator. If you get graphics artifacts, consider enabling software
rendering with "--enable-software-rendering".
Launching lib\main.dart on AOSP on IA Emulator in debug mode...
[!] Your app isn't using AndroidX.
    To avoid potential build failures, you can quickly migrate your app by following the steps on https://goo.gl/CP92wY.

Compiler message:
../AppData/Roaming/Pub/Cache/hosted/pub.dartlang.org/cached_network_image-1.1.3/lib/src/cached_network_image_provider.dart:42:24: Error: The method 'CachedNetworkImageProvider.load' has fewer positional arguments than those of overridden method 'ImageProvider.load'.
  ImageStreamCompleter load(CachedNetworkImageProvider key) {
                       ^
/C:/Program%20Files/Flutter/flutter/packages/flutter/lib/src/painting/image_provider.dart:403:24: Context: This is the overridden method ('load').
  ImageStreamCompleter load(T key, DecoderCallback decode);
                       ^
Target kernel_snapshot failed: Exception: Errors during snapshot creation: null
build failed.

FAILURE: Build failed with an exception.

* Where:
Script 'C:\Program Files\Flutter\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 780

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\Program Files\Flutter\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 10s
Running Gradle task 'assembleDebug'...
Running Gradle task 'assembleDebug'... Done                        11.1s
Gradle task assembleDebug failed with exit code 1
```

**Comment by chris75898** [comment]
Found a breaking change with the newer version of [package flutter_cached_network_image](https://github.com/renefloor/flutter_cached_network_image). 

Updating pubspec.yaml resolves this issue.

`cached_network_image: 2.0.0-rc`

**Comment by jorgeribeiro** [comment]
> Found a breaking change with the newer version of [package flutter_cached_network_image](https://github.com/renefloor/flutter_cached_network_image).
> 
> Updating pubspec.yaml resolves this issue.
> 
> `cached_network_image: 2.0.0-rc`

It worked for me, thanks.
You should open a PR with this update.

**Comment by jfschmakeit** [comment]
We have a fix for this ready and should be pushing out an update to the codelab next week. Please stay tuned!

(There was also a breaking change with the library dependency that required an additional change.)

**Comment by jfschmakeit** [comment]
This has been fixed by commit 628c81e14b0e14f688e527de9bb95ed898ad7bd3.
